frontend/src/components/VaultDetails.vue

```diff
@@ -202,18 +213,19 @@ import FetchError from './FetchError.vue';
 import GrantPermissionDialog from './GrantPermissionDialog.vue';
 import ReactivateVaultDialog from './ReactivateVaultDialog.vue';
 import RecoverVaultDialog from './RecoverVaultDialog.vue';
-import RecoveryKeyDialog from './RecoveryKeyDialog.vue';
+import DisplayRecoveryKeyDialog from './DisplayRecoveryKeyDialog.vue';
 import SearchInputGroup from './SearchInputGroup.vue';
 
 const { t, d } = useI18n({ useScope: 'global' });
 
 const props = defineProps<{
   vaultId: string,
-  role: VaultRole
+  vaultRole: string,
```

Comment:
Let's restrict this type to the set of allowed values:

```suggestion
  vaultRole: 'OWNER' | 'MEMBER' | 'NONE',
```